charts/policies/allow-privilege-escalation-psp/questions.yaml

```diff
@@ -0,0 +1,63 @@
+# This is a Rancher questions file
+---
+
+# General policy questions:
+questions:
+- variable: "metadata.name"
+  type: string
+  default: allow-privilege-escalation-psp
+  required: true
+  description: Name of policy resource
+  group: Settings
+  label: Name of policy resource
```

Comment:
I think a better pattern for this is using `nameOverride` and `fullnameOverride` as values.yaml fields, as done in the upstream `kube-prometheus-stack` chart: https://github.com/prometheus-community/helm-charts/blob/88447d1886a982b0e26f047d9973d2f6bad7fe7e/charts/kube-prometheus-stack/templates/_helpers.tpl#L7-L25

Also don't think this needs to be exposed in the values.yaml; ideally, you should be using the name of the release that this is deployed as to name the resources.

Reply:
Good point.
Made it use `.Release.name` as `.metadata.name` of the policy resources, non-configurable, in https://github.com/kubewarden/helm-charts/pull/131/commits/52672972caa8963a6f59f985f4dc5ebabce1f950.